packages/@aws-cdk/aws-lambda-nodejs/test/integ.dependencies.ts

```diff
@@ -24,6 +27,40 @@ class TestStack extends Stack {
   }
 }
 
+class SdkV3TestStack extends Stack {
```

Comment:
I think integ testing can place in `https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-lambda-nodejs/test/integ.function.ts`. WDYT.
And run testing like this way https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-lambda-nodejs/test/integ.function.ts

Reply:
[integ.function.ts](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-lambda-nodejs/test/integ.function.ts) tests which can the stack be deployed simply.
And [integ.dependencies.ts](https://github.com/yamatatsu/aws-cdk/blob/externalModules-for-sdkV3/packages/@aws-cdk/aws-lambda-nodejs/test/integ.dependencies.ts) has been testing about esbuild `externalModules` option before I edited. And I think no needed to change this division of responsibility.

I've added a test to confirm which the lambda function can be invoked without runtime error. And I believe this addition doesn't change the division of responsibility.